src/components/Widgets/Widget.tsx

```diff
@@ -200,13 +197,7 @@ export function Widget({ starterVariant, themeVariant }: WidgetProps) {
     >
       {isMultisigSigner && <MultisigWalletHeaderAlert />}
       <ClientOnly
-        fallback={
-          <WidgetSkeleton
-            welcomeScreenClosed={welcomeScreenClosed}
-            // title={starterVariant === 'default' ? 'Exchange' : 'Gas'}
-            // buttonTitle={starterVariant === 'default' ? 'Connect' : 'Get Gas'}
-          />
-        }
+        fallback={<WidgetSkeleton welcomeScreenClosed={welcomeScreenClosed} />}
       >
         <LiFiWidget integrator={config.integrator} config={config} />
```

Comment:
why not directly integrate the LiFiWidget inside the blog rather than reusing this component?